docs/source/installation/for_dev.rst

```diff
@@ -72,9 +72,9 @@ Installing Manim using Poetry
          poetry install
 
     .. note:: The first time running this command, poetry will create and
-              enter a virtual environment rooted at the current directory.
```

Comment:
debatable...

Reply:
I'd also prefer "at" to "in", but only weakly.

Reply:
Ok, so I'm leaving it then